internal/locale/locales/en-us.yaml

```diff
@@ -1976,6 +1976,13 @@ svcctl_wait_timeout:
     Please try running your command again.
 
     If this problem persists please report it on our forums: [ACTIONABLE]{{.V2}}[/RESET]
+svcctl_file_not_found:
+  other: |
+    Could not find the State Tool service executable.
+
+    Please try running your command again.
+
+    If this problem persists please reinstall the State Tool.
```

Comment:
I think mentioning the forums here would be better. Do we know for a fact that reinstalling would resolve these issues?

Reply:
Fair point and no we don't, reinstall could end up with the same result.